qa/tasks/vstart_runner.py

```diff
@@ -758,7 +758,7 @@ def list_connections():
                                   self.mountpoint]
 
         self.fuse_daemon = self.client_remote.run(args=fuse_cmd_args,
-                                                  wait=False)
+                                                  wait=False, omit_sudo=False)
```

Comment:
@lxbsz

Reply:
Okay, right, I just forgot the vstart_runner.py is using its own mount methods.